src/components/Table/Table.tsx

```diff
@@ -3,17 +3,19 @@ import * as React from 'react';
 import * as Rivet from '../util/Rivet';
 
 interface TableProps {
-    /** 
+    /**
      * Optional Rivet style which changes the look and feel of the rendered table.
      */
-    variant?: 'stripes' | 'plain'
+    variant?: 'stripes' | 'plain' | 'compact' | 'cells'
```

Comment:
Are these variants actually mutually exclusive? From the documentation, it sounds like either could be applied to a striped or plain table (or alongside each other).

Reply:
@burnumd That's a good call. Yes, they design so that they can be applied alongside each other if needed.